lib/automation.go

```diff
@@ -210,6 +210,9 @@ func (automationImpl) Apply(scope scope, p *ApplyParams) (*ApplyResult, error) {
 			return nil, err
 		}
 	}
+	if err := scope.Logbook().ProfileCanWrite(scope.Context(), ref.InitID, scope.ActiveProfile()); err != nil {
```

Comment:
can we get at least one test that proves this denies access locally?

Reply:
oh, right... mb rushing stuff lately.